app/routes/_index.tsx

```diff
@@ -1,16 +1,69 @@
-import { type MetaFunction } from '@remix-run/node'
+import HeroCallToAction from '#app/components/organisms/Hero/HeroCallToAction.tsx'
+import { Button } from '#app/components/ui/button.tsx'
+import { json, useLoaderData, Link, type MetaFunction } from '@remix-run/react'
+import heroImage from '~/assets/jpg/sample-hero.jpg'
+import ArticleCard from '~/components/organisms/ArticleCard.tsx'
+import { prisma } from '~/utils/db.server.ts'
 
 export const meta: MetaFunction = () => [{ title: 'Epic News' }]
 
+export async function loader() {
+	const allArticles = await prisma.article.findMany({
+		select: {
+			id: true,
+			title: true,
+			category: { select: { name: true } },
+			images: { select: { id: true } },
+		},
+	})
+
+	return json({ allArticles })
+}
 export default function Index() {
+	const { allArticles } = useLoaderData<typeof loader>()
+
 	return (
-		<main className="grid h-full place-items-center">
-			<h1 className="text-mega">
-				Hello from{' '}
-				<pre className="prose rounded-lg bg-primary p-6 text-primary-foreground">
-					app/routes/_index.tsx
-				</pre>
-			</h1>
+		<main className="  grid  h-full  place-items-center ">
+			<HeroCallToAction
+				image={heroImage}
+				imageRight={true}
+				hasBackgroundColour={true}
+			>
+				<div className="flex h-full flex-1 flex-col justify-between p-16">
+					<div className="flex flex-col gap-8">
+						<h2 className="text-h2">Welcome to Epic News</h2>
+						<p className="text-lg">
+							Keep up to date with the latest tech news.
+						</p>
+					</div>
+					<Button asChild variant="default" size="lg">
+						<Link to="/signup">Sign up</Link>
+					</Button>
+				</div>
+			</HeroCallToAction>
+
+			<div className="container py-16">
+				<h2 className="mb-8 text-h2 font-normal">Latest news</h2>
+
+				<div className="grid grid-cols-2 gap-6 md:grid-cols-3 lg:grid-cols-4">
+					{allArticles.length > 0 ? (
+						allArticles.map(article => (
+							<ArticleCard
+								key={article.id}
```

Comment:
Hey Rich - you've forgotten to pass in the `articleId` from the `article` being mapped :).

Just add it in with the other props and you'll fix the bug:

```tsx
<ArticleCard
	key={article.id}
	articleId={article.id}
	title={article.title}
	category={article.category?.name}
	imageId={article.images[0]?.id}
/>
```